lib/crux.ig/deps.edn

```diff
@@ -1,6 +1,7 @@
 {:paths ["src"]
  :deps
  {juxt/crux-core {:mvn/version "19.07-1.3.0-alpha"}
+  juxt/crux-rocksdb {:mvn/version "19.07-1.3.0-alpha"}
```

Comment:
Shouldn't this be in the demo application, and crux.ig has no opinion about how you load in your crux?